R/datasets.R

```diff
@@ -617,6 +617,25 @@
 #' \item{year}{Year of Sale}
 #' \item{region}{Region Where Sale Took Place}
 #' }
-"avocados" 
+"avocados"
+
+
+#' House Prices and Properties in Saratoga, New York
+#' 
+#' Random sample of 1057 houses taken from full Saratoga Housing Data (De Veaux)
+#' @format A data frame with 1057 observations on the following 8 variables
+#' \describe{
+#' \item{price}{Price (US dollars)} 
+#' \item{living.area}{Living Area (square feet)}
```

Comment:
```
> checking for code/documentation mismatches ... WARNING
  Data codoc mismatches from documentation object 'saratoga_houses':
  Variables in data frame 'saratoga_houses'
    Code: Age Bathrooms Bedrooms Fireplace Fireplaces Living.Area
          Lot.Size Price
    Docs: age bathrooms bedrooms fireplace fireplaces living.area
          lot.size price
```
@heschmidt you have to re-run the script file and commit the changes to `data/saratoga_houses.rda`. Note that the variable names should have underscores and not `.`.

Reply:
@heschmidt @alejanmg @egordonhalpern we still need to fix this piece.